### PR TITLE
Update Test cases

### PR DESCRIPTION
**Pull request by ruihan00**
## Status
In progress

## Description
Update testcases to fit the new model and commands

## Checklist
- [x] update entry testcases
- [x] update storage testcases
- [x] update expenditure testcases
- [ ] update command testcases

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/AddCommandIntegrationTest.java [hunk 1]
```diff
@@ -22,7 +25,25 @@ public class AddCommandIntegrationTest {
...
-        model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+        model = new ModelManager(getTypicalPennyWise(), new UserPrefs());
+    }
+
+    @Test
+    public void execute_newExpenditure_success() {
+        Entry validExpenditure = new ExpenditureBuilder().build();
+        Model expectedModel = new ModelManager(model.getPennyWise(), new UserPrefs());
+        expectedModel.addExpenditure(validExpenditure);
+        assertCommandSuccess(new AddCommand(validExpenditure, new EntryType("e")), model,
```
Comment (EmilyOng) [inline src/test/java/seedu/address/logic/commands/AddCommandIntegrationTest.java:36]:
```suggestion
        assertCommandSuccess(new AddCommand(validExpenditure, new EntryType(EntryType. ENTRY_TYPE_EXPENDITURE)), model,
```

I think it's better if we use the defined constants

File: src/test/java/seedu/address/logic/commands/AddCommandIntegrationTest.java [hunk 2]
```diff
@@ -22,7 +25,25 @@ public class AddCommandIntegrationTest {
...
+                String.format(AddCommand.MESSAGE_SUCCESS, validExpenditure), expectedModel);
+    }
+
+    @Test
+    public void execute_duplicateExpenditure_throwsCommandException() {
+        Entry expenditureInList = model.getPennyWise().getExpenditureList().get(0);
+        assertCommandFailure(
+                new AddCommand(expenditureInList, new EntryType("e")),
```
Comment (EmilyOng) [inline src/test/java/seedu/address/logic/commands/AddCommandIntegrationTest.java:44]:
```suggestion
                new AddCommand(expenditureInList, new EntryType(EntryType. ENTRY_TYPE_EXPENDITURE)),
```

File: src/test/java/seedu/address/model/PennyWiseTest.java [hunk 3]
```diff
@@ -36,56 +46,91 @@ public void resetData_null_throwsNullPointerException() {
...
+        assertThrows(NullPointerException.class, () -> pennyWise.hasExpenditure(null));
+    }
+
+    @Test
+    public void hasIncome_nullIncome_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> pennyWise.hasIncome(null));
+    }
+
+    @Test
+    public void hasExpenditure_expoenditureNotInAddressBook_returnsFalse() {
```
Comment (EmilyOng) [inline src/test/java/seedu/address/model/PennyWiseTest.java:76]:
```suggestion
    public void hasExpenditure_expenditureNotInPennyWise_returnsFalse() {
```

File: src/test/java/seedu/address/model/PennyWiseTest.java [hunk 4]
```diff
@@ -36,56 +46,91 @@ public void resetData_null_throwsNullPointerException() {
...
+
     // @Test
-    // public void hasPerson_personInAddressBook_returnsTrue() {
-    //     addressBook.addPerson(ALICE);
-    //     assertTrue(addressBook.hasPerson(ALICE));
+    // public void hasIncome_incomeNotInAddressBook_returnsFalse() {
+    //     assertFalse(pennyWise.hasIncome());
     // }
+    @Test
+    public void hasExpenditure_expenditureInAddressBook_returnsTrue() {
```
Comment (EmilyOng) [inline src/test/java/seedu/address/model/PennyWiseTest.java:85]:
```suggestion
    public void hasExpenditure_expenditureInPennyWise_returnsTrue() {
```

File: src/test/java/seedu/address/model/entry/EntryTypeTest.java [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+public class EntryTypeTest {
+
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new EntryType((null)));
+    }
+
+    @Test
+    public void constructor_invalidEntryTYpe_throwsIllegalArgumentException() {
```
Comment (EmilyOng) [inline src/test/java/seedu/address/model/entry/EntryTypeTest.java:17]:
```suggestion
    public void constructor_invalidEntryType_throwsIllegalArgumentException() {
```

File: src/test/java/seedu/address/storage/JsonUserPrefsStorageTest.java [hunk 6]
```diff
@@ -49,31 +49,30 @@ private Path addToTestDataPathIfNotNull(String userPrefsFileInTestDataFolder) {
...
+        UserPrefs expected = getTypicalUserPrefs();
+        UserPrefs actual = readUserPrefs("ExtraValuesUserPref.json").get();
+        assertEquals(expected, actual);
+    }
 
     private UserPrefs getTypicalUserPrefs() {
         UserPrefs userPrefs = new UserPrefs();
         userPrefs.setGuiSettings(new GuiSettings(1000, 500, 300, 100));
-        userPrefs.setPennyWiseFilePath(Paths.get("addressbook.json"));
+        userPrefs.setPennyWiseFilePath(Paths.get("data/addressbook.json"));
```
Comment (EmilyOng) [inline src/test/java/seedu/address/storage/JsonUserPrefsStorageTest.java:75]:
```suggestion
        userPrefs.setPennyWiseFilePath(Paths.get("data/pennywise.json"));
```

**Review by EmilyOng — COMMENTED** [review]
Thanks! LGTM! Should I make a PR to your fork for the remaining test cases? Or we can merge this first after clearing up

File: src/test/java/seedu/address/logic/commands/AddCommandIntegrationTest.java (hunk 1 above)
Comment (ruihan00) [reply src/test/java/seedu/address/logic/commands/AddCommandIntegrationTest.java:36]: good point!

**Pull request merged**